LSPS6/README.md

```diff
@@ -0,0 +1,130 @@
+# LSPS6
+
+| Name    | `.well-known`      |
+| ------- |--------------------|
+| Version | 1                  |
+| Status  | For Implementation |
```

Comment:
Initial Status should be [draft](https://github.com/BitcoinAndLightningLayerSpecs/lsp?tab=readme-ov-file#status-specification).